src/main/java/hudson/plugins/collabnet/util/Helper.java

```diff
@@ -65,30 +73,37 @@ public static String getToken(URL ctfUrl, String ctfUserName, String ctfPassword
         return token;
     }
 
-    public static String getWebhookToken(String webhookUrl, String webhookUsername, String webhookPassword,
-                                         TaskListener listener)
-            throws
-            IOException {
+    public static String getWebhookUrl(String ctfUrl) throws IOException {
+        String webhookUrl = null;
         CloseableHttpClient client = null;
         CloseableHttpResponse response = null;
-        String token = null;
+        String webhookEndPoint = null;
         try {
-            client = CNFormFieldValidator.getHttpClient();
-            HttpPost httpPost = new HttpPost(webhookUrl);
-            StringEntity entity = new StringEntity(getLoginData(webhookUsername, webhookPassword).toString());
-            httpPost.setEntity(entity);
-            httpPost.setHeader("Accept", "application/json");
-            httpPost.setHeader("Content-type", "application/json");
-            response = client.execute(httpPost);
-            if(response.getStatusLine().getStatusCode() == 200) {
-                token = JSONObject.fromObject(EntityUtils.toString(response.getEntity())).getJSONObject("Response").
-                        get("SessionToken").toString();
+            webhookUrl =  ctfUrl.concat(":3000/publishers/v4");
+            client = HttpClients.createDefault();
```

Comment:
Use 'CNFormFieldValidator.getHttpClient()' whenever you need to create a http client, so that ignoreSslErrors flag is taken into account.

Reply:
Done.